.github/workflows/publish-cdk-command.yml

```diff
@@ -74,9 +73,9 @@ jobs:
         uses: mariamrf/py-package-publish-action@v1.1.0
         with:
           # specify the same version as in ~/.python-version
-          python_version: '3.7.9'
-          pip_version: '21.1'
-          subdir: 'airbyte-cdk/python/'
+          python_version: "3.9.11"
```

Comment:
is it worth standardising on 3.9.11 everywhere?

Reply:
I think I have. Did I miss one?
Or are you saying not to change this one? I was following the hint on line 76 there.

Reply:
I was wondering out loud whether it's worth standardising the entire repo to 3.9.11 instead of having 3.9.11 here and 3.9 on the connectors.